pom.xml

```
@@ -64,7 +64,7 @@
     </scm>
 
     <properties>
-        <rewrite.version>7.35.0</rewrite.version>
+        <rewrite.version>7.36.0</rewrite.version>
```

Were you looking to keep this open until after the 7.36.0 release? Otherwise we can test it now with
```suggestion
        <rewrite.version>7.36.0-SNAPSHOT</rewrite.version>
```